rewards/divvi-integrators/1745107200000-1747243524718/safe-transactions.json

```
@@ -0,0 +1,36 @@
+{
+  "meta": {},
+  "transactions": [
+    {
+      "to": "0x326161d68c05bE55367a0041b9C8f68082C04863",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address[]",
+            "name": "users",
+            "type": "address[]"
+          },
+          {
+            "internalType": "uint256[]",
+            "name": "amounts",
+            "type": "uint256[]"
+          },
+          {
+            "internalType": "uint256[]",
+            "name": "rewardFunctionArgs",
+            "type": "uint256[]"
+          }
+        ],
+        "name": "addRewards",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "users": "[0xb82896c4f251ed65186b416dbdb6f6192dfaf926]",
+        "amounts": "[200000000]",
+        "rewardFunctionArgs": "[1745107200000, 1747243524718]"
```

Because the underlying block timestamp precision is up to a second, we should preferably call the script with 0 milliseconds.
```suggestion
        "rewardFunctionArgs": "[1745107200000, 1747243524718]"
```

---

ah, k, thanks, i'll update the internal docs too.